code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -425,6 +425,11 @@
 /datum/species/proc/can_understand(var/mob/other)
 	return
 
+/datum/species/proc/adjustBruteLoss(var/mob/living/carbon/C, var/amount)
+	return 1
+/datum/species/proc/adjustBurnLoss(var/mob/living/carbon/C, var/amount)
+	return 1
```

Comment:
Why is this handled on the species?

Reply:
Because there was no way to handle species specific, non-static damage mods.